web_m2x_options/readme/CONTRIBUTORS.rst

```diff
@@ -4,8 +4,8 @@
 * Nicolas JEUDY <nicolas@sudokeys.com>
 * Yannick Vaucher <yannick.vaucher@camptocamp.com>
 * Zakaria Makrelouf <z.makrelouf@gmail.com>
-* `Tecnativa <https://www.tecnativa.com>`_:
-
-  * Jairo Llopis <jairo.llopis@tecnativa.com>
-  * David Vidal <david.vidal@tecnativa.com>
-  * Ernesto Tejeda <ernesto.tejeda87@gmail.com>
+* Tecnativa <https://www.tecnativa.com>
+* Jairo Llopis <jairo.llopis@tecnativa.com>
+* David Vidal <david.vidal@tecnativa.com>
+* Ernesto Tejeda <ernesto.tejeda87@gmail.com>
```

Comment:
This was indented because all these contributors belong to Tecnativa. Please leave it as it was.